tests/generators/run_generators_in_browser.js

```diff
@@ -40,14 +40,18 @@ async function runLangGeneratorInBrowser(browser, filename, codegenFn) {
 async function runGeneratorsInBrowser() {
   var options = {
     capabilities: {
-      browserName: 'firefox'
+      browserName: 'chrome',
```

Comment:
Update the comments for this function to not say that it's using firefox.

Looking forward, should this entire file/set of functions move into gulp scripts? Is there any reason we should keep it as a separate thing? It looks kind of like it went manual testing -> bash script -> node script.

Reply:
Done, and perhaps? I'm not familiar enough with the tooling to have a useful opinion, but simplifying/using the standard approach SGTM.